File: code/community/1301706051602616330/payment-decline-resolution.py
import os
from deepgram import Deepgram

# Set your Deepgram API key in environment variable
api_key = os.getenv('DEEPGRAM_API_KEY')

dg_client = Deepgram(api_key)

source = {"url": "https://storage.googleapis.com/deepgram-demos/voicemail.mp3"}

response = dg_client.transcription.prerecorded(source, {
  'model': 'general',
  'language': 'en-US'
})

print(response)

File: code/community/1301706051602616330/payment-decline-resolution.cs
using Deepgram;
using System;
using System.Threading.Tasks;

class Program
{
    static async Task Main(string[] args)
    {
        var deepgramApiKey = Environment.GetEnvironmentVariable("DEEPGRAM_API_KEY");
        var client = new DeepgramClient(deepgramApiKey);

        var source = new { url = "https://storage.googleapis.com/deepgram-demos/voicemail.mp3" };

        var response = await client.Transcription.PreRecordedAsync(source, new { language = "en-US" });

        Console.WriteLine(response);
    }
}